From 490ddff0958221e8c402cd4c87c37f9acc2c92f4 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Thu, 2 Apr 2015 14:14:00 +0800
Subject: [PATCH] MLK-10558 ARM: dts: imx6qp-sabreauto: enable PRG2 support

The previous commit e8e43b960723f0cae129327f392761a57269bcc9 has
disabled PRG2 by accident. This patch restored it.

Signed-off-by: Robby Cai <r63905@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index 128411f..793477d 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -37,6 +37,7 @@
 
 &prg2 {
 	memory-region = <&memory>;
+	status = "okay";
 };
 
 &sata {
-- 
1.8.0

